server/src/Middleware/ConnectionManagerMiddleware.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Middleware;
+
+use App\Service\ConnectionService;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Message\ResponseInterface;
+
+class ConnectionManagerMiddleware {
+	/**
+	 * @var ConnectionService
+	 */
+	private $connService;
+
+	/**
+	 * @param ConnectionService $connService
+	 */
+	public function __construct( ConnectionService $connService ) {
+		$this->connService = $connService;
+	}
+
+	/**
+	 * @param ServerRequestInterface $request
+	 * @param ResponseInterface $response
+	 * @param callable $next
+	 * @return ResponseInterface
+	 */
+	public function __invoke(
```

Comment:
I don't understand what this method/middleware is doing... maybe add a description to the docblock explaining it's purpose?